sample-apps/donet-sample-app/collector.yaml

```diff
@@ -0,0 +1,20 @@
+#collector.yaml in the root directory
```

Comment:
can this file be deleted and we keep only the first one?

Reply:
I agree as the spec is same for all sample apps